src/Control/LocaleAdmin.php

```diff
@@ -36,6 +38,21 @@ protected function init()
         Requirements::css("tractorcow/silverstripe-fluent:client/dist/styles/fluent.css");
     }
 
+    public function getEditForm($id = null, $fields = null)
+    {
+        $form = parent::getEditForm($id, $fields);
+
+        // Add sortable field to locales
+        if ($this->modelClass === Locale::class) {
+            /** @var GridField $listField */
+            $listField = $form->Fields()->dataFieldByName($this->sanitiseClassName($this->modelClass));
+            $config = $listField->getConfig();
+            $config->addComponent(new GridFieldOrderableRows('Sort'));
```

Comment:
Is this already a dependency of fluent?

Reply:
Yep `"symbiote/silverstripe-gridfieldextensions": "^3.1",`